Index: Lessons/src/Task7.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Lessons/src/Task7.java	(date 1581784479002)
+++ Lessons/src/Task7.java	(date 1581784479002)
@@ -0,0 +1,29 @@
+public class Task7 {
+
+    /*Имеется два дома размерами a на b и c на d. Имеется участок размерами e на f.
+    Проверить помещаются ли эти дома на данном участке. Стороны домов - параллельны
+    сторонам участка, в остальном размещение может быть любым.*/
+
+    public static void main(String[] args) {
+            /*Ввод параметров участка и домов*/
+        int HPlot = 15;
+        int WPlot = 15;
+        int HFirstHouse = 5;
+        int WFirstHouse = 12;
+        int HSecondHouse = 8;
+        int WSecondHouse = 4;
+            /*Первая проверка входят ли дома по площади на участок*/
+        if (HPlot * WPlot >= (HFirstHouse * WFirstHouse) + (HSecondHouse * WSecondHouse)){
+            /*Проверяем поместятся ли дома до одной из сторон участка*/
+            if (HPlot >= HFirstHouse + HSecondHouse && WPlot >= WSecondHouse + WFirstHouse){
+                System.out.println("Дома поместятся на участок");
+                } else if (HPlot >= HFirstHouse + WSecondHouse && WPlot >= WFirstHouse + HSecondHouse){
+                System.out.println("Дома поместятся на участок");
+                } else {
+                System.out.println("Дома на учаток НЕ поместятся");
+                }
+            } else {
+            System.out.println("Дома на учаток НЕ поместятся");
+        }
+    }
+}
Index: Lessons/src/Task6.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Lessons/src/Task6.java	(date 1581780437129)
+++ Lessons/src/Task6.java	(date 1581780437129)
@@ -0,0 +1,29 @@
+public class Task6 {
+
+    /*Имеются три числа - день, месяц, год.
+    Ввести в виде трех чисел дату следующего дня.*/
+
+    public static void main(String[] args) {
+        int Day = 31;               //Будем считать что в каждом месяце 31 день, а по хорошему
+        int Month = 12;             //нужно выщитывать каждый месяц и смотреть на год.
+        int Year = 2020;            //Но наверняка для всего этого уже есть разработанная функция.
+        /*Проверка на корректность даты*/
+        if (Day < 1 || Day > 31) {
+            System.out.println("Введите день от 1 до 31");
+        } else if (Month < 0 || Month > 12) {
+            System.out.println("Введите месяц от 1 до 12");
+        } else {
+            Day = Day + 1;      //Увеличение даты на один день
+            if (Day > 31) {
+                Month = Month + 1;
+                Day = 1;
+                if (Month > 12) {
+                    Year = Year + 1;
+                    Month = 1;
+                }
+                System.out.println("Следующая дата: " + Day + "." + Month + "." + Year);
+
+            }
+        }
+    }
+}
